components/browser/icons/src/main/java/mozilla/components/browser/icons/BrowserIcons.kt

```diff
@@ -79,7 +79,7 @@ class BrowserIcons(
     jobDispatcher: CoroutineDispatcher = Executors.newFixedThreadPool(THREADS).asCoroutineDispatcher()
 ) {
     private val logger = Logger("BrowserIcons")
-    private val maximumSize = context.resources.pxToDp(MAXIMUM_SIZE_DP)
+    private val maximumSize = context.resources.displayMetrics.pxToDp(MAXIMUM_SIZE_DP)
```

Comment:
This code doesn't look right. I understand that this is a fairly mechanical refactor. 

The function `pxToDp` would appear to take a `px` value and spit out a `dp` value. This line appears to give that function a `dp`.

I think `git blame` and ask whoever wrote this line originally.

Reply:
#754 is where it was decided on. I'll ask about the name.

Reply:
May I suggest API like:

```kotlin
private val maximumSize = MAXIMUM_SIZE_DP.pxToDpWith(context)
```

?